packages/webhook/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@slack/webhook",
-  "version": "7.0.3",
+  "version": "7.1.0",
```

Comment:
We might be alright to keep this as a patch bump after landing [the dependency bump](https://github.com/slackapi/node-slack-sdk/milestone/69?closed=1)?

Reply:
I often put package updates as patch changes but please let me know if `semver:minor` is the way here 🙏

Reply:
This one is a bit weird because it's a dependency bump, but also supposed to also resolve a security vuln 🤔 But I guess that vuln is considered a bug, so perhaps patch makes more sense here?

Reply:
I have pondered this and agree that patch makes more sense, especially when thinking in terms of dependabot PRs. I'll update the webhook milestone!

Reply:
✨ 🐛 🔍  I like to think a `security` vuln. makes it a fancy patch?

I was thinking about @dependabot too and was also curious how updates have been in past minor vs. patch changes, but [`@slack/webhook`](https://www.npmjs.com/package/@slack/webhook?activeTab=versions) doesn't hint at much here!